src/mock/utils/fnr-utils.ts

```diff
@@ -51,4 +61,54 @@ function getFiresifretÅr(fødselsnummer: string) {
         return year + 1900;
     }
 
-}
\ No newline at end of file
+}
+
+function getRiktigKjønnPåFødslesnummer(fødselsnummer: string, kjønn: Kjønn) {
+    if (utledKjønnFraFødselsnummer(fødselsnummer) !== kjønn) {
+        const tilfeldigPartall = getTilfeldigPartall().toString();
+        const tilfeldigOddetall = (getTilfeldigPartall() + 1).toString();
+        const kjønnsiffer = kjønn === Kjønn.Kvinne ? tilfeldigPartall : tilfeldigOddetall;
+
+        var nyttfnr = fødselsnummer.substr(0, 8) + kjønnsiffer + fødselsnummer.substr(9, 11);
+        nyttfnr = nyttfnr.substr(0, 9) + getKontrollsiffer1(nyttfnr) + nyttfnr.substr(10, 11);
+        nyttfnr = nyttfnr.substr(0, 10) + getKontrollsiffer2(nyttfnr);
+
+        return nyttfnr;
+    } else {
+        return fødselsnummer;
+    }
+}
+
+function getTilfeldigPartall() {
+    return (Math.floor(Math.random() * 4) + 1) * 2;
```

Comment:
Her kunne det vært nais om vi bruke den seeda faker-instansen til å lage tilfeldige tall. For å få samme data hver gang vi søker opp et fødselsnummer.

Det er ikke noe big-deal, men jeg synes det er litt kult at vi har deterministiske test-personer. F.eks at hvis man oppdager en bug på informasjonen tilknyttet en person, så er bugen der ved neste page-last også.